src/internal.c

```diff
@@ -8176,132 +8176,271 @@ void DtlsTxMsgListClean(WOLFSSL* ssl)
     ssl->dtls_tx_msg_list = head;
 }
 
-/* Create a DTLS Fragment from *begin - end, adjust new *begin and bytesLeft */
-static DtlsFrag* CreateFragment(word32* begin, word32 end, const byte* data,
-                                byte* buf, word32* bytesLeft, void* heap)
+static DtlsFragBucket* CreateFragBucket(word32 offset, const byte* data,
```

Comment:
Consider using a common prefix for all DtlsMsg functions

Reply:
Changing to use the common prefix.